src/molecules/Footer.tsx

```diff
@@ -133,6 +133,9 @@ const StyledContainerNewsletter = styled.div`
       .breakpoints.md.maxWidth}) {
     text-align: center;
   }
+  @media (max-width: ${Layout.breakpoints.sm.maxWidth}) {
```

Comment:
If you update to `atomic-layout@0.16.0` you will be able to shorten this media query declaration to:

```js
import { query } from 'atomic-layout'

const StyledContainerNewsletter = styled.div`
  @media ${query({ to: 'sm' })} {
    margin-bottom: -20px;
  }
`
```

Also, consider using `Box` component here, so you can control `marginBottom` responsively from props:

```jsx
<Box as={StyledContainerNewsletter} marginBottomSmDown="-20px" />
```

Choose whichever option you find the most suitable.